pylint_odoo/misc.py

```diff
@@ -158,6 +158,22 @@ def check_js_lint(self, fname):
                 output.strip('\n').split('\n')[:-2])
         return output_spplited
 
+    def check_po_lint(self, fname):
+        """Check po lint in fname.
+        :param fname: String with full path of file to check
+        :return: Return list of errors.
+        """
+        cmd = ['dennis-cmd', 'lint', '--reporter=line', fname]
```

Comment:
If you are using a python package try use the library with `import dennis` and get all list of messages directly to avoid use `subprocess`